### PR TITLE
Egyptian Clothing, Egyptian Null Rod, Craftable Mummy Costume!

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
Sprites by Onule [IDK his github name]

The staff is only available icon wise and will have tangible features later down the line as I fix up archaeology.

Clothing not available normally yet, save that for Halloween or archaeology, whichever comes first.

### CODE REVIEW

**Comment by Armhulen** [comment]
is it an exosuit
because we already have mummy clothing, make sure it works WITH the mummy clothes

**Comment by DoctoreMad** [comment]
You can check the files changed and SEE if its a exosuit

**Comment by DaxDupont** [comment]
Please add screenshots!

**Comment by ExcessiveUseOfCobblestone** [comment]
> we already have mummy clothes

pardon?

**Comment by pubby** [comment]
The mummy sprites we already have:

image

I prefer this new headgear but they're both good quality.

**Comment by DoctoreMad** [comment]
Look at his .gif. His .gif is amazing.

**Comment by Armhulen** [comment]
make the new headgear replace the pharoh old one, make the clothes themselves an exosuit so the mummy goes under

**Comment by ExcessiveUseOfCobblestone** [comment]
the headgear goes well with the dark colors of the mummy costume though but yeah ill turn it into an exosuit.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -441,6 +441,14 @@
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	allowed = list(/obj/item/clothing/mask/facehugger/toy)
 
+/obj/item/clothing/suit/nemes
```
Comment (DoctoreMad) [inline code/modules/clothing/suits/miscellaneous.dm]: I keep reading this as "memes" please fix

Comment (ExcessiveUseOfCobblestone) [reply code/modules/clothing/suits/miscellaneous.dm]: I can't fix stupid

**Comment by Supermichael777** [comment]
so mummy head-wrap as a balaclava and mummy wraps as a shirt(both craftable) with this stuff as a suit and helmet so you can go around as a mumy cosplayer/fake mob(***GET OUT***)?

**Comment by meeeeme** [comment]
WE

**Comment by FuryMcFlurry** [comment]
Im ok with replacing my shitty pharaoh helm with this new sick one

**Comment by FantasticFwoosh** [comment]
Yes the new proposed headpiece is also made to fit comfortably on your exosuit too, im not sure if the old one matches up ok, or simply would be less detail for the same effect if it does match up.

- Could you post a mummy with bandages, (new) headgear and the exosuit to show us when you get the chance?

**Comment by Repukan** [comment]
WE

**Comment by ExcessiveUseOfCobblestone** [comment]
There's nothing wrong with both and I think the older ones color scheme works better with the mummy clothes and Onule's works better with the tunic. I'm keeping both headdresses.

> WE
WUZ

**Comment by pubby** [comment]
We could change the old mummy's color palette to be a bit brighter.

**Comment by KorPhaeron** [comment]
@ExcessiveUseOfCobblestone should add the staff as a chaplain thing

**Comment by ExcessiveUseOfCobblestone** [comment]
hahahaha omg yes!

**Comment by ExcessiveUseOfCobblestone** [comment]
> you can have commit summaries be emojis

this is the future.

**Comment by KorPhaeron** [comment]
I'll merge it as is if you dont want to add any more, but your staff desc made me think that a recipe that consumes a stack of bandages and requires the staff as a tool to make the mummy costume would be a fun perk for it to have

**Comment by ExcessiveUseOfCobblestone** [comment]
hahahaha you're a trip

File: code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -558,3 +558,17 @@
 	tools = list(/obj/item/weldingtool, /obj/item/screwdriver, /obj/item/wrench)
 	reqs = list(/obj/item/stack/sheet/metal = 15)
 	category = CAT_MISC
+
+/datum/crafting_recipe/mummy/
```
Comment (Fox-McCloud) [inline code/modules/crafting/recipes.dm]: no `/` at the end, here, please.

**Pull request merged**